activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   The SQLite3 adapter now implements the `supports_deferrable_constraints?` contract
+
+    Implementing the full `supports_deferrable_constraints?` contract allows foreign keys to be deferred by adding the `:deferrable` key to the `foreign_key` options in the `add_reference` and `add_foreign_key` methods.
```

Comment:
Can you wrap this to ~80ish characters please? :bow:

Reply:
Done in https://github.com/rails/rails/pull/49376/commits/38dd5f41396c30f50e83d5858e84a09db81aa09c